build-conflicts.php

```diff
@@ -190,91 +123,57 @@ static function (array $components): array {
                 $conflicts[$component->name->packageName] = $component->getConflictConstraint();
             }
 
-            ksort($conflicts);
-
-            return array_filter($conflicts);
+            return Dict\filter(Dict\sort_by_key($conflicts));
         };
 
     $buildConflictsJson = static function (array $baseConfig, array $conflicts): string {
-        return json_encode(
-            array_merge(
-                $baseConfig,
-                ['conflict' => $conflicts]
-            ),
-            JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE
-        );
+        return Json\encode(Dict\merge($baseConfig, ['conflict' => $conflicts]), true);
```

Comment:
`JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE` are always used, `JSON_PRETTY_PRINT` is set via the `$pretty` argument ( true )